src/main/java/JarvisException.java

```diff
@@ -0,0 +1,6 @@
+public class JarvisException extends Exception{
+    @Override
+    public String getMessage() {
+        return "OOPS!!! ";
```

Comment:
Perhaps leaving the whitespace for the inherited classes would be clearer? One would have to be cognisant that the parent class already has a whitespace; and this isn't consistent with how other Java exceptions are implemented.

```suggestion
        return "OOPS!!!";
```